apps/api/e2e/tests/rewards.test.ts

```diff
@@ -17,26 +17,31 @@ import { RewardsService, createLeafNode } from "@/services/rewards.service.js";
 
 describe("Rewards Service", () => {
   let rewardsService: RewardsService;
-  let testEpochId: string;
+  let testCompetitionId: string;
 
   beforeEach(async () => {
     rewardsService = new RewardsService();
 
-    // Create a test epoch with UUID
-    const epochId = "756fddf2-d5a3-4d07-b769-109583469c88";
-    const [epoch] = await db
-      .insert(epochs)
+    // Create a test competition with UUID
+    const competitionId = "756fddf2-d5a3-4d07-b769-109583469c88";
+    const [competition] = await db
+      .insert(competitions)
       .values({
-        id: epochId,
-        startedAt: new Date(),
+        id: competitionId,
+        name: "Test Competition",
+        description: "A test competition for rewards",
+        status: "active",
+        type: "trading",
+        startDate: new Date(),
+        endDate: new Date(Date.now() + 7 * 24 * 60 * 60 * 1000), // 7 days from now
       })
       .returning();
 
-    if (!epoch) {
-      throw new Error("Failed to create test epoch");
+    if (!competition) {
```

Comment:
We have been trying to clean up tests, and there is a PR that landed yesterday, so you will probably have conflicts in this file.
  
In general, this should be an assertion instead of a conditional that throws, i.e. 
```
expect(competition).toBeDefined();
expect(competition?.id).toBe(competitionId);
```  
Also, any `console.log`s should be removed.